docs/development_submitting.md

```diff
@@ -42,4 +42,4 @@ Submit your improvements, fixes, and new features to Home Assistant one at a tim
     - Type a title and complete the provided template for your pull request.
     - Click **Create pull request**.
 
-7. Check for comments and suggestions on your pull request and keep an eye on the [CI output](https://github.com/home-assistant/core/actions).
+7. Check for comments and suggestions on your pull request and keep an eye on the [CI output](https://github.com/home-assistant/core/actions). If this is your first time submitting a pull request, the CI won't run until a maintainer approves running the CI. Just wait, a maintainer will eventually come by and approve it.
```

Comment:
```suggestion
7. Check for comments and suggestions on your pull request and keep an eye on the [CI output](https://github.com/home-assistant/core/actions).

:::info
If this is your first time submitting a pull request, the CI won't run until a maintainer approves running it. Just wait, a maintainer will eventually come by and approve it.
:::
```
Would wrap this into a info block, this way it is a bit more visible. Wording wise I leave this for someone other.

Reply:
<img width="975" alt="image" src="https://github.com/home-assistant/developers.home-assistant/assets/3989428/6962cd3e-d1a9-4168-9d05-b3b1baac145c">